CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Change Log
+## [Unreleased]
+## [0.0.14] - 2017-08-03
```

Comment:
maybe we need to sync this changelog with debian changelog? or should we maintain these separated? I'd rather the first option.